### DISCUSSION TITLE
Pass additional parameters to job processor (separate file)

### DISCUSSION THREAD

**Discussion by borkowsky** [OptimalBits/bull#2707] [Q&A]
In my application, there was a need to share a common instance between JOB process files. Simply connecting the application initialization file is not suitable, since this does not save the settings table (I use ExpressJS); therefore, it is necessary to separate the common instance of ExpressJS App. How can I implement this?

**Comment by OwaliShawon** [comment]
To share a common instance of ExpressJS between multiple files or processes, you can create a module that exports the initialized Express app. This way, you can import this module in different files or processes, ensuring they all share the same instance of the Express app.

**Comment by borkowsky** [comment]
> To share a common instance of ExpressJS between multiple files or processes, you can create a module that exports the initialized Express app. This way, you can import this module in different files or processes, ensuring they all share the same instance of the Express app.

App variables, setted up by another files not accessible in job process files. Result is undefined.
